r/README.md

```diff
@@ -23,7 +24,17 @@ Windows or macOS.
 install.packages("arrow")
 ```
 
-Alternatively, if you are using conda you can install arrow from conda-forge:
+Alternatively, you can install the development version of arrow from R-universe.
+
+```r
+# Binaries for MacOS/Windows
+install.packages("arrow", repos = "https://apache.r-universe.dev")
+
+# Binaries for Linux (Ubuntu 24.04)
+install.packages("arrow", repos = "https://rpolars.r-universe.dev/bin/linux/noble/4.4")
```

Comment:
Probably a copying error.

As it is, I would not recommend using the binary URL for Ubuntu in R-universe unless you are familiar with the internals of R-universe.
The installation will fail unless you understand what the differences in Ubuntu versions mean.
e.g. etiennebacher/tidypolars#137

Reply:
> Probably a copying error.

Ah, good catch.

> As it is, I would not recommend using the binary URL for Ubuntu in R-universe unless you are familiar with the internals of R-universe.

Fair enough. I'll remove and link to the [description](https://docs.r-universe.dev/install/binaries.html) of Ubuntu binaries in the r-universe docs.